modin/pandas/test/test_groupby.py

```diff
@@ -167,19 +167,19 @@ def test_mixed_dtypes_groupby(as_index):
         )
         eval_general(modin_groupby, pandas_groupby, lambda df: df.ewm(com=0.5).std())
         eval_shift(modin_groupby, pandas_groupby)
-        eval_mean(modin_groupby, pandas_groupby)
+        eval_mean(modin_groupby, pandas_groupby, numeric_only=True)
```

Comment:
No. I use this option explicitly to test the same as before. In the previous release of pandas, this parameter was determined automatically.

Reply:
Makes sense. However I think we should also test for `numeric_only=False` (most likely in a separate issue, as I would presume it would incur some bugfixing - I would be very surprised if we don't have any bugs on an area we don't test!)

Reply:
https://github.com/modin-project/modin/issues/6241